documentation/docs/05-load.md

```diff
@@ -99,12 +99,14 @@ An instance of [`URL`](https://developer.mozilla.org/en-US/docs/Web/API/URL), co
 
 #### depends
 
-This function declares that the `load` function has a _dependency_ on one or more URLs, which can subsequently be used with [`invalidate()`](/docs/modules#$app-navigation-invalidate) to cause `load` to rerun.
+This function declares that the `load` function has a _dependency_ on one or more URLs or custom identifiers, which can subsequently be used with [`invalidate()`](/docs/modules#$app-navigation-invalidate) to cause `load` to rerun.
 
 Most of the time you won't need this, as `fetch` calls `depends` on your behalf — it's only necessary if you're using a custom API client that bypasses `fetch`.
 
 URLs can be absolute or relative to the page being loaded, and must be [encoded](https://developer.mozilla.org/en-US/docs/Glossary/percent-encoding).
 
+Custom identifiers have to be prefixed with one or more lowercase letters followed by a colon to conform to the [URI specification](https://www.rfc-editor.org/rfc/rfc3986.html)
```

Comment:
This is brilliant! Had been agonising over what sort of API would work for both URLs and non-URLs, and all this time you can just do `a:b`. This makes life much easier

Reply:
It has been in the description of https://github.com/sveltejs/kit/pull/4536 all this time. But I should have documented it a bit clearer.

I can relate to your agony. It took me a lot of thinking and tinkering with the `URL` constructor to arrive at this seemingly obvious solution.